_config/protected.yml

```diff
@@ -10,3 +10,6 @@
 - /video/youtube/i1aiUaa8RZ8/
 - /video/youtube/LbzkY9nrgdo/
 - /video/youtube/Q37niG_hCcA/
+- /video/youtube/pIURFD15WJ0/
+- https://plus.google.com/events/cmoaedv5ib92rlhh9bnjrkd1hhg?authkey=CNnNnODpxtTzbQ
+- /webinars/net-on-rhel-sneak-peek
```

Comment:
I believe this is the landing page. We shouldn't protect this page, as we want to use this to encourage users to register.